spec/features/viewing_a_meeting_spec.rb

```diff
@@ -25,11 +25,11 @@
       visit meeting_path(meeting)
       login_mock_omniauth(member, 'Log in')
 
-      click_on 'Attend'
+      click_on 'RSVP here'
 
       expect(page).to have_content('Your RSVP was successful. We look forward to seeing you at the Monthly!')
 
-      click_on "Can't make it anymore? Click here to cancel your spot."
```

Comment:
@KimberleyCook any reason why we removed the period from this sentence? I think I prefer it with period.

Reply:
Yeah I removed them as I think they looked odd 😂